### PR TITLE
Fix a few typos and small problems on the code search cheat sheet page

### PR DESCRIPTION
**Pull request by vdavid**
I was just reading the page to learn about the search syntax, spotted a few problems, and decided to fix them to improve this page.

For the types, I used https://docs.sourcegraph.com/code_search/reference/queries as a source.

## Type of change
Typo fixes.

### What should this PR do?
There's no related Asana or Jira issue. These are random minor fixes to a page content, not requested by anyone.

### Why are we making this change?
I intend to improve this page.

### What are the acceptance criteria? 
The page should not look broken. The content should be better than it was before the change.

### How should this PR be tested?
- Check out the GitHub diff (I think the two-column view is best for this), see if you spot any typos in my version.
- If there's a test environment to run this branch then go to that environment, scroll through the page and see the new content, and make sure it looks good.

## Pull request process

**Reviewers**:

1. Test functionality using the criteria above.
2. Offer tips for efficiency, feedback on best practices, and possible alternative approaches and things that may not have been considered.
3. For shorter, "quick" PRs, use your best judgement on #​2.
4. Use a collaborative approach and provide resources and/or context where appropriate.
5. Provide screenshots/grabs where appropriate to show findings during review.

**Reviewees**:

1. Prefer incremental and appropriately-scoped changes.
2. Leave a comment on things you want explicit feedback on.
3. Respond clearly to comments and questions.

### CODE REVIEW

File: posts/how-to-search-code-with-sourcegraph-a-cheat-sheet.md [hunk 1]
```diff
@@ -128,7 +128,7 @@ after:time-period'
 matcher='time-period'
 /> 
 
-Sometimes the time period can be literal, like `last week`, `last year`, `3 months ago`, `february 10 2021` or have actual dates in the format `dd/mm/yyyy`.
+Sometimes the time period can be relative, like `last week`, `last year`, `3 months ago` or absolute, in several formats including `{month} {day} {year}` (example: `february 10 2021`), `dd/mm/yyyy`, and ISO format `yyyy-mm-dd`.
```
Comment (ltagliaferri) [inline posts/how-to-search-code-with-sourcegraph-a-cheat-sheet.md:131]: I thought not everyone may be familiar with ISO so added "format" for context, feel free to contextualize more

Comment (ltagliaferri) [reply posts/how-to-search-code-with-sourcegraph-a-cheat-sheet.md:131]: Also "relative" is a good call

File: posts/how-to-search-code-with-sourcegraph-a-cheat-sheet.md [hunk 2]
```diff
@@ -13,9 +13,9 @@ type: posts
 
 Sourcegraph is a universal code search tool, enabling you to search across both open source and your own private code repositories. Code search can help you onboard onto new codebases, contribute to open source, find bugs and error messages, understand dependency libraries, and more.
 
-This cheat sheet style guide can help you get up to speed with Sourcegraph commands quickly. For more thorough tutorials on using Sourcegraph, refer to our tutorials(https://learn.sourcegraph.com/tags/sourcegraph) and our documentation(https://docs.sourcegraph.com/). 
+This cheat sheet style guide can help you get up to speed with Sourcegraph commands quickly. For more thorough tutorials on using Sourcegraph, refer to our [tutorials](https://learn.sourcegraph.com/tags/sourcegraph) and our [documentation](https://docs.sourcegraph.com/).
```
Comment (ltagliaferri) [inline posts/how-to-search-code-with-sourcegraph-a-cheat-sheet.md:16]: I noticed the links were not formatted correctly, so fixed them here

**Review by ltagliaferri — APPROVED** [review]
Thank you for the update, everything looks good. I piggybacked with the markdown formatting when I realized they were not right in reviewing what you changed. I resolved conversations so you were not blocked, but just wanted added "format" behind "ISO" to give context to those not familiar with the initials, and said "relative" was a good language change, in addition to the corrections.

Feel free to merge

**Comment by vdavid** [comment]
Thank you, @ltagliaferri!
After creating the PR, I realized you were not with Sourcegraph anymore.
So, special thanks for jumping in! :)

**Pull request merged**